### PR TITLE
Timescale backend docs

### PR DESCRIPTION
**Pull request by c0c0n3**
This PR updates the manual to cater for the new Timescale backend, see #233.

### CODE REVIEW

File: docs/manuals/admin/timescale.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Timescale
+
+[Timescale][timescale] is one of the time series databases that can be
+used with QuantumLeap as a back end to store NGSI entity time series.
+Indeed, QuantumLeap provides full support for storing NGSI entities in
+Timescale, including geographical features (encoded as GeoJSON or NGSI
+Simple Location Format), structured types and arrays. Moreover, it is
+possible to dynamically select, at runtime, which storage back end to
+use (Crate or Timescale) depending on the tenant who owns the entity
```
Comment (chicco785) [inline docs/manuals/admin/timescale.md]: i would not cover discussion for multiple support here. better in index or some other generic place https://github.com/smartsdk/ngsi-timeseries-api/blob/master/docs/manuals/index.md 
index should be updated as well

File: docs/manuals/admin/timescale.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+* `POSTGRES_DB_NAME`: the name of the QuantumLeap DB, defaults to
+  `quantumleap`.
+* `POSTGRES_DB_USER`: the DB user QuantumLeap should use to connect,
+  defaults to `quantumleap`.
+* `POSTGRES_DB_PASS`: the above user's password, defaults to `*`.
+* `POSTGRES_USE_SSL`: should QuantumLeap connect to PostgreSQL using
+  SSL? If so, then set this variable to any of: `true`, `yes`, `1`, `t`.
+  Specify any other value or don't set the variable at all to use a
+  plain TCP connection.
+* `QL_CONFIG`: absolute pathname of the QuantumLeap YAML configuration
```
Comment (chicco785) [inline docs/manuals/admin/timescale.md]: this should be discussed in a generic doc not in the timescale specific one

File: docs/manuals/admin/timescale.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+    default-backend: Crate
+
+With this configuration, any NGSI entity coming in for tenant `t1`
+or `t3` will be stored in Timescale whereas tenant `t2` will use
+Crate. Any tenant other than `t1`, `t2`, or `t3` gets the default
+Crate back end.
+
+
+## Migrating from Crate
```
Comment (chicco785) [inline docs/manuals/admin/timescale.md]: may be better here https://github.com/smartsdk/ngsi-timeseries-api/blob/master/docs/manuals/admin/dataMigration.md

in this doc i would link to it

Comment (c0c0n3) [reply docs/manuals/admin/timescale.md]: yep, good idea. done :-)

**Comment by c0c0n3** [comment]
> beyond comments inline, we should also update:

whoops, yes, sorry I missed that. Done :-)
see d6f61e2

**Comment by chicco785** [comment]
also main index.md needs updates

**Comment by c0c0n3** [comment]
> also main index.md needs updates

yes, I revamped it. see 05058d3

File: docs/manuals/admin/timescale.md (hunk 2 above)
Comment (c0c0n3) [reply docs/manuals/admin/timescale.md]: moved to own section, see ff09f87

File: docs/manuals/admin/timescale.md (hunk 1 above)
Comment (c0c0n3) [reply docs/manuals/admin/timescale.md]: moved content to landing page, see 05058d3 and 0b8bab4

**Pull request merged**